rfc/5/versions/1/index.md

```diff
@@ -0,0 +1,977 @@
+
+
+##### <a name=trafo-byDimension>byDimension</a>
+
+`byDimension` transformations build a high dimensional transformation
+using lower dimensional transformations on subsets of dimensions.
+The `input` and `output` fields MUST always be included for this transformations type.
+
+<dl>
+  <dt><strong>transformations</strong></dt>
+  <dd>  Each child transformation MUST contain <code>input_axes</code> and <code>output_axes</code> fields
+        whose values are arrays of strings.
+        Every axis name in a child transformation's <code>input_axes</code>
+        MUST correspond to a name of some axis in this parent object's <code>input</code> coordinate system.
+        Every axis name in the parent byDimension's <code>output</code> coordinate system
+        MUST appear in exactly one child transformation's <code>output_axes</code> array.
+        Each child transformation's <code>input_axes</code> and <code>output_axes</code> arrays
+        MUST have the same length as that transformation's parameter arrays.
+        </dd>
+</dl>
+
+##### <a name="bijection">bijection</a>
+
+A bijection transformation is an invertible transformation in
+which both the `forward` and `inverse` transformations are explicitly defined.
+Each direction SHOULD be a transformation type that is not closed-form invertible.
+Its input and output spaces MUST have equal dimension.
+The input and output dimensions for the both the forward and inverse transformations
+MUST match bijection's input and output space dimensions.
+
+`input` and `output` fields MAY be omitted for the `forward` and `inverse` transformations,
+in which case the `forward` transformation's `input` and `output` are understood to match the bijection's,
+and the `inverse` transformation's `input` (`output`) matches the bijection's `output` (`input`),
+see the example below.
+
+The `input` and `output` fields MAY be omitted for `bijection` transformations
+if the fields may be omitted for both its `forward` and `inverse` transformations
+
+Practically, non-invertible transformations have finite extents,
+so bijection transforms should only be expected to be correct / consistent for points that fall within those extents.
+It may not be correct for any point of appropriate dimensionality.
+
+### "multiscales" metadata
+
+Metadata about an image can be found under the `multiscales` key in the group-level OME-Zarr Metadata.
+Here, "image" refers to 2 to 5 dimensional data representing image
+or volumetric data with optional time or channel axes.
+It is stored in a multiple resolution representation.
+
+`multiscales` contains a list of dictionaries where each entry describes a multiscale image.
+
+Each `multiscales` dictionary MUST contain the field "coordinateSystems",
+whose value is an array containing valid coordinate system metadata
+(see [coordinate systems](#coordinatesystems-metadata)).
+The last entry of this array is the "default" coordinate system
```

Comment:
```suggestion
The last entry of this array is the "base" coordinate system
```
I got quite confused here, because this is the same coordinate system that all the multiscale arrays are transformed into, but not the "default" coordinate system for the multiscales (that is the first entry of the array) described above. So I suggest changing this to "base" to avoid confusion.

Reply:
Yes, exactly. "intrinsic" is a good name 👍